app/views/layouts/application.html.erb

```diff
@@ -6,13 +6,33 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
- [ ] The screenshot below shows the error I encountered when I tried testing the application. I think it has to do with tailwindcss configuration. I will suggest you look into it at your earliest convenience.

Reply:
|Error Screenshot |
|<img width="1204" alt="Screen Shot 2022-09-30 at 4 27 15 PM" src="https://user-images.githubusercontent.com/39523312/193316841-f5711729-3bfd-412a-9853-f5024d65eb8c.png">|

Reply:
Thank you for providing the feedback.
please follow the below setup process to make everything works fine
### Install & Setup

To setup and install this sample AlFa Blog project, follow the below steps:
- Clone this project by the command: 
  ```
  git clone https://github.com/aliatayee/AlFaBlog
  ```

- Then switch to the project folder by the bellow query:

  ```
  cd AlFaBlog
  ```

- Install the required dependecies with:
  ```
  bundle install
  ```
- Install npm dependencies with: 
  ```
  npm install
  ```
- Before running below commands update [config/database.yml](./config/database.yml) with your local `DB`credentials. Then run:
    ```
    rake db:create
    ```
    ```
    rake db:migrate
    ```
    ```
    rake db:seed
    ```
- To make the tailwindcss works run the below command:
    ```
    bundle exec rake assets:precompile
    ```

- With Ruby on Rails and PostgreSQL installed in your environment, you can run `rails server` to run the website.